sktime/benchmarking/__init__.py

```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+"""Implements running validations on estimators storing results for benchmarking."""
+from kotsu import registration
+
+estimator_registry = registration.ModelRegistry()
```

Comment:
if these registries are forecasting - specific should they be named as such? e.g. forecasting_estimators or sthg. otherwise they give a misleading impression of being global